swift/Tests/Blockchains/AvalancheTests.swift

```diff
@@ -0,0 +1,99 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import WalletCore
+import XCTest
+
+class AvalancheTests: XCTestCase {
+
+    func testAddress() {
+        let key = PrivateKey(data: Data(hexString: "0x98cb077f972feb0481f1d894f272c6a1e3c15e272a1658ff716444f465200070")!)!
+        let pubkey = key.getPublicKeySecp256k1(compressed: false)
+        let address = AnyAddress(publicKey: pubkey, coin: .avalancheXChain)
+        let addressFromString = AnyAddress(string: "X-avax1apmh7wxg3js48fhacfv5y9md9065jxuft30vup", coin: .avalancheXChain)!
+
+        XCTAssertEqual(pubkey.data.hexString, "0x02b33c917f2f6103448d7feb42614037d05928433cb25e78f01a825aa829bb3c27")
```

Comment:
should be `04b33c917f2f6103448d7feb42614037d05928433cb25e78f01a825aa829bb3c27299a9fb02dd081d37876f145b8d3aa47fe9764145a9d90d4f941105d7de85c82`, uncompressed public key